OptionSelector/OSStuffTableViewController.m

```diff
@@ -0,0 +1,59 @@
+//
+//  OSStuffTableViewController.m
+//  OptionSelector
+//
+//  Created by Daniel Distant on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OSStuffTableViewController.h"
+
+@interface OSStuffTableViewController ()
+
+@end
+
+@implementation OSStuffTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+
+    return self.category.options.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"stuffCell" forIndexPath:indexPath];
+    
+    cell.textLabel.text = [self.category.options objectAtIndex:indexPath.row];
+    
+    return cell;
+}
+
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    [tableView cellForRowAtIndexPath:indexPath].accessoryType = UITableViewCellAccessoryCheckmark;
+    
+    
```

Comment:
You're missing something important here. If you look at `CQCategory.h`, you'll see a `selection` property. You could set its value here as follows:
```objc
self.category.selection = self.category.options[indexPath.row];
```
Because your master view controller also has access to the same category object (the one it passed to this view controller in `prepareForSegue:`, that line implicitly passes data from your detail VC to the master VC.